### ISSUE TITLE
Missing transaction in a lot of item

### ISSUE THREAD

**Issue by JessicaSOS** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#418]
**Already been postponed on February 16, 2018**

There are transactions that do not appear in the history. Each inventory adjustment should appear to be a positive or negative adjustment, via a tiket, an inventory transfert, an RMA, a manual adjustment, etc...

**item #2459**
How is it possible to pass the 0 to 1 (so there are 1 in inventory) and the next transaction, to go from 0 to 2. If there were 1 in inventory, how can we get to 0 without have made any other transactions? See attachment
<img width="938" alt="item 2459" src="https://user-images.githubusercontent.com/40500094/44237488-a4f0dd00-a17e-11e8-8ad9-6e1b19d252ac.png">

**item #9139**
Missing transactions ... see attachment
<img width="932" alt="item 9139" src="https://user-images.githubusercontent.com/40500094/44237492-aae6be00-a17e-11e8-8f47-762fd0b29535.png">

--
French:
Déjà été reporté le 16 février 2018

Il y a des transactions qui n'apparaissent pas dans l'historique. Chaque ajustement d'inventaire devrait apparaitre que se soit un ajustement positif ou négatif, via un tiket, un transafert d'inventaire, un RMA, un ajustement manuel, etc...

**Item #2459**
Comment est-ce possible de passer le 0 à 1 (il y en a donc 1 en inventaire) et la transaction suivant, de passer de 0 à 2. S'il y en avait 1 en inventaire, comment pouvons nous arrivé à 0 sans avoir fait aucune autre transaction? Voir pièce jointe

**item #9139** 
Il manque des transactions... voir pièce jointe

**Comment by usmanalikhan93** [comment]
Hi @JessicaSOS

Sorry for the inconvenience.
We are reviewing this issue internally. Once I get something on this, I will circle back to you.

Usman Ali.

**Comment by JessicaSOS** [comment]
any news ? that this still produce ....

**Comment by usmanalikhan93** [comment]
Hi @JessicaSOS

I have come up with one use case after testing, this issue comes is case of e.g. when you are logged in on store A and in inventory transfer you select from store B and to store C, it does not log history against that inventory. So for now I recommend you that please select from store in which you are logged in.

Furthermore, I am discussing this with developer for log transfer history in this case. I will update you when I get more info on this.

Usman Ali

**Comment by usmanalikhan93** [comment]
Hi @JessicaSOS 

I hope you are doing well. We have released a fix for this issue as I have mentioned the use case earlier. Now history will be come up in inventory in these cases. I hope in future, you will not face these issues again related to missing transactions. 

For now I am closing all tasks you reported related to missing inventory.

**Comment by JessicaSOS** [comment]
There is no A-B-C-D store. There is an error in the store inventory. There are missing transactions. See example attached since the changes were supposedly done on your side (2019) ...
Please open this task again, it is not fixed.
<img width="956" alt="9148" src="https://user-images.githubusercontent.com/40500094/51621369-5fd98f80-1f02-11e9-9ca7-f29f2668fc37.png">

--
Il n'y a pas de magasin A-B-C-D. Il y a une erreur dans l'inventaire des magasins. Il manque des transactions. Voir exemple en pièce jointe depuis que les modifications ont supposément été faite de votre côté (2019)...
Merci d'ouvrir à nouveau cette tâche, ce n'est pas corrigé.

**Comment by JessicaSOS** [comment]
I agree to close task # 407 and # 432 since this is the same problem. However, as I wrote to you recently in task # 418, you mentioned that this has been fixed since 2019. I returned you an example in 2019 of an item that had the same problem. This is not corrected. Thank you for reopening the task.

--
french:
Je suis d'accord a fermé la tâche #407 et #432 puisqu'il s'agit du même problème. Cependant, comme je vous ai écrit dernièrement dans la tâche #418, vous m'aviez mentionné que cela était corrigé depuis l'an 2019. Je vous ai retourné un exemple en 2019 d'un item qui avait le même problème. Ce n'est donc pas corrigé. Merci de réouvrir la tâche.
